proto/prehog/v1/teleport.proto

```diff
@@ -67,9 +67,68 @@ message UserActivityRecord {
   uint64 kube_port_sessions = 13;
 }
 
+// the kind of a "resource" (e.g. a node, a pod, a service, etc.)
```

Comment:
Nit - this looks like a minor deviation in this comment from the copy in the cloud repo.